.github/ISSUE_TEMPLATE/focus-area-proposal.yml

```diff
@@ -1,19 +1,29 @@
-name: Proposal
-description: Proposal for feature to include in Interop 2023
-labels: proposal
+name: Focus Area Proposal
+description: Proposal for an Interop 2023 Focus Area
+labels: focus-area-proposal
 body:
 - type: markdown
   attributes:
     value: |
-      **Interop 2023 has not yet launched and this documentation is still being finalized. Please do not submit proposals yet.**
- 
       Thanks for taking the time to make a proposal for Interop 2023!
 
       Please see the [Interop 2023 README](https://github.com/web-platform-tests/interop/blob/main/2023/README.md) for the timeline and what to expect from the process.
+
+      If this feature does not have a high quality specification and tests, an [Investigation Effort](https://github.com/web-platform-tests/interop/issues/new?template=investigation-effort-proposal.yml) may be more appropriate.
```

Comment:
This sounds like something which is early in the specification process might be suitable for an Investigate effort. We probably want to say "is already widely implemented, but lacks a high quality specification or tests" or similar.

Reply:
I took that wording almost verbatim.